src/rpc/blockchain.cpp

```diff
@@ -2696,6 +2696,9 @@ class CoinsViewScanReserver
 
 static RPCHelpMan scantxoutset()
 {
+    // scriptPubKey corresponding to mainnet address 12cbQLTFMXRnSzktFkuoG3eHoMeFtpTu3S
```

Comment:
> 
  ~Note that the example descriptor used doesn't follow the developer guideline of using invalid bech32 addresses, as the RPC is not wallet-related and it's use-case is merely to look up state information (i.e. there is no danger of sending funds to a wrong address).~ For the sake of simplicity, the raw descriptor for an early coinbase payout address (block 9) is taken, i.e. it yields results even at an early stage of IBD. Happy to change that though if there are other suggestions.


Should it be a valid dash address?

Reply:
meh? @UdjinM6

Reply:
yeah, why not.

```
> dash-cli deriveaddresses 'raw(76a91411b366edfc0a8b66feebae5c2e25a7b6a5d1cf3188ac)#fm24fxxy'
[
  "XcJSEb79KEeNbwMU7eE27aL5dhDwvjgBuH"
]
```

Reply:
```suggestion
    // scriptPubKey corresponding to mainnet address XcJSEb79KEeNbwMU7eE27aL5dhDwvjgBuH
```